### DISCUSSION TITLE
PointerPressed event stop in ScrollContentPresenter

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#11780] [Q&A] [answered]
I have a grid under the ScrollViewer
```
<Grid>
    ...
<Grid/>
...
<ScrollViewer Name="ScrollViewer1">
                   ...
</ScrollViewer>
```
And the Grid will not trigger any pointer event, how to trigger to Grid?
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/a6d83dc8-b934-4b19-95dd-8faa90412430)

**Comment by timunie** [comment]
I guess the grid need to have a background set. Transparent is okay, but not null which is the default.

**Reply by Coloryr** [reply]
is set

**Reply by Coloryr** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/b3c9c166-dbf9-493a-bdff-c03dc88420ae)

**Reply by Coloryr** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/03f72c7b-e97b-4e27-855a-8e92e9359b8f)

**Reply by Coloryr** [reply]
```
<ScrollViewer Name="ScrollViewer1" IsHitTestVisible="False">
```
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/6ed15476-1429-4ecc-8da6-c65b1b700705)

**Reply by Coloryr** [reply]
If remove ScrollViewer is will trigger.

**Reply by timunie** [reply]
Can you try to add a tunnel event? this may get the grid triggered first. https://docs.avaloniaui.net/misc/wpf/tunnelling-events

**Reply by Coloryr** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/6a9b2c00-0143-4ac4-ae71-37d076626417)
Is look stop in `ToggleButton`

**Reply by Coloryr** [reply]
Is add here?
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/48fb5d3c-a51e-4437-a01e-01171a595a0a)
The target grid name is `Live2dTop`

**Reply by Coloryr** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/4ab033fc-b102-456a-a1ba-6b27c72e4e84)

**Reply by Coloryr** [reply, marked as answer]
```
<ScrollViewer Name="ScrollViewer1" VerticalAlignment="Top">
```
Is work now
